main.js

```diff
@@ -0,0 +1,143 @@
+        origin: 'United States',
+        imageUrl: 'https://goo.gl/Cro3tL'
+      },
+      {
+        itemId: 7,
+        name: 'Bodhran',
+        brand: 'Meinl',
+        price: 129.99,
+        description: 'A frame drum for people who like to jig.',
+        details: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.',
+        origin: 'Ireland',
+        imageUrl: 'https://goo.gl/YNb74O'
+      },
+      {
+        itemId: 8,
+        name: 'TR-09',
+        brand: 'Roland',
+        price: 399,
+        description: 'A drum machine for people with no desk space.',
+        details: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.',
+        origin: 'Japan',
+        imageUrl: 'https://goo.gl/gJYUxz'
+      }
+    ]
+  }
+}
+
+function renderItem(catalogItem, number) {
+  let searchCatalog = catalogItem.catalog.items
+  let $item =
+    createElement('div', {class: 'card col-lg-2 border-success mt-4 pt-4'}, [
+      createElement('img', {class: 'card-img-top', style: 'max-height: 10rem', src: searchCatalog[number].imageUrl, alt: 'image cap'}, []),
+      createElement('div', {class: 'card-body text-center d-flex flex-column'}, [
+        createElement('h5', {class: 'mt-auto card-title'}, [searchCatalog[number].brand]),
+        createElement('p', {class: 'card-text'}, [searchCatalog[number].name]),
+        createElement('p', {class: 'card-text text-success font-weight-bold'}, [searchCatalog[number].price])
+      ])
+    ])
+  return $item
+}
+
+function renderAllItems(allItems) {
+  let $container = createElement('div', {class: 'container mt-3'}, [])
+  let $header = createElement('h1', {class: 'font-weight-light'}, ['Jamazon'])
+  let $row = createElement('row', {class: 'row'}, [])
+  let searchCatalog = allItems.catalog.items
+  for (let i = 0; i < searchCatalog.length; i++) {
+    $row.appendChild(renderItem(allItems, i))
```

Comment:
Instead of passing all items to `renderItem`, just pass one, so it doesn't need to figure out which one to render.